TeamCode/src/main/java/org/firstinspires/ftc/teamcode/DriveTrain.java

```diff
@@ -0,0 +1,246 @@
+package org.firstinspires.ftc.teamcode;
+
+import android.util.Log;
+
+//import com.qualcomm.hardware.bosch.BNO055IMU;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.Gamepad;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
+import org.firstinspires.ftc.robotcore.external.navigation.Orientation;
+
+public class DriveTrain {
+    String DRIVE_TRAIN_CAPTION = "Drive Status";
+    Telemetry telemetry;
+    HardwareInnov8Robot robot;
+    LinearOpMode opMode;
+
+    //Hi, I'm adding this comment so I'll have made a change to my branch (TeleopDrivetranMechanum)
+
+    double wheelOnePower = 0.0; // 0.55 // The standard power for the wheels, will probably be changed later
+    double wheelTwoPower = 0.0; // 0.7
+    double wheelThreePower = 0.0; //0.35
+    double wheelFourPower = 0.0; // 0.2
+    double wheelSpeedChanger = 1.1;
+    double inchToTick = (360/6); // The number of encoder ticks per inch for our wheels
```

Comment:
for constants like this, it is convention to make it `public static final` with a big shouty name
ex: `public static double INCH_TO_TICK = (360/6)`